### PR TITLE
MAT-3965

### PR DESCRIPTION
**Pull request by adongare**
## PR Template

Jira Ticket: [MAT-3965 ](https://jira.cms.gov/browse/MAT-3965 )
(Optional) Related Tickets:

### Summary
This PR is for:
- Conversion of MAT measure to MADiE models
- Transfer MAT measure to MADiE
- Test cases

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e. Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [x] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [x] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e. Changing auth methods, Adding a new user type, Adding a required but vulnerable package)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: lambda-function/src/models/Measure.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Model } from "./Model";
```
Comment (adongare) [inline lambda-function/src/models/Measure.ts:1]: this will be replaced by madie-models npm package.

**Review by gregory-akins — APPROVED** [review]
